app/HieWrapper.hs

```diff
@@ -73,15 +73,13 @@ run opts = do
   logm $ "Current directory:" ++ d
   logm $ "Operating system:" ++ os
 
-  -- Get the cabal directory from the ghc-mod cradle
-  (mcr,_) <- GM.runGhcModT GM.defaultOptions GM.cradle
-  dir <- case mcr of
-    Left err -> error (show err)
-    Right cr -> return $ GM.cradleRootDir cr
+  -- Get the cabal directory from the cradle
+  cradle <- findLocalCradle (d </> "File.hs")
```

Comment:
Unsure this will work well with multi-cradle projects.

Reply:
It wont, we have to tackle it in hie-bios

Reply:
I manually tested and documented how I intend it to work in https://github.com/mpickering/haskell-ide-engine/blob/hie-bios/hie-plugin-api/Haskell/Ide/Engine/Cradle.hs#L256

Reply:
So, this will work in a best effort manner, imo.